crates/rolldown_plugin/src/plugin_driver/build_hooks.rs

```diff
@@ -110,8 +118,22 @@ impl PluginDriver {
         .instrument(debug_span!("resolve_id_hook", plugin_name = plugin.call_name().as_ref()))
         .await?
       {
+        trace_action!(action::HookResolveIdCallEnd {
```

Comment:
Again, can we reconsider RAII way? This Introduce too much noise for source code.

Reply:
Yes. I haven't figured out how to deal with different fields that `Start` and `End` action have.